tests/test_utils.py

```diff
@@ -54,28 +54,25 @@ def test_guess_by_bom(encoding, expected):
 
 
 def test_bad_get_netrc_login():
-    assert get_netrc_login("url") is None
-
     os.environ["NETRC"] = "tests/.netrc"
-    assert get_netrc_login("url") is None
-
-    os.environ["NETRC"] = "wrongpath"
-    assert get_netrc_login("url") is None
+    assert str(get_netrc()) is not None
 
     from httpx import utils
 
     utils.NETRC_STATIC_FILES = ()
+
+    os.environ["NETRC"] = "wrongpath"
+    assert utils.get_netrc() is None
+
     os.environ["NETRC"] = ""
-    assert utils.get_netrc_login("url") is None
+    assert utils.get_netrc() is None
 
 
 def test_get_netrc_login():
     os.environ["NETRC"] = "tests/.netrc"
-    assert get_netrc_login("netrcexample.org") == (
-        "example-username",
-        None,
-        "example-password",
-    )
+    netrc_content = str(get_netrc())
+    assert "example-username" in netrc_content
+    assert "example-password" in netrc_content
```

Comment:
The previous test was more precise in the assertion, I think we can stick to this style:

```python
netrc = get_netrc()
assert netrc.authenticators("netrcexample.org") == (
    "example-username", None, "example-password",
)
```